container/root/run.d/03-debug.sh

```diff
@@ -7,6 +7,9 @@ then
   FASTCGI_BUFFER_COUNT=4
   FASTCGI_BUFFER_SIZE=16k
 
+  # IMPORTANT: avoids error "must be equal to or greater than the maximum of the value of 'proxy_buffer_size' and one of the 'proxy_buffers'"
+  FASTCGI_BUSY_BUFFER=4M
```

Comment:
Woah, thank you for that.  I couldn't pin that one down when I encountered it...